src/patternfly/components/Masthead/masthead.scss

```diff
@@ -173,6 +185,22 @@ $pf-c-masthead--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl
     display: flex;
     align-items: center;
   }
+
+  .pf-c-dropdown__toggle {
```

Comment:
If these style changes only apply to the full height variation, do you think it makes sense to add `.pf-m-full-height` to the selectors?

Also curious what you think about making these border changes the default in the component variations, assuming if you're using a full height toggle, you don't want the top border, and maybe you want the larger bottom border width?

Can I see these changes in use in this PR or do we need to merge https://github.com/patternfly/patternfly/pull/4108 first? This is what the current demo looks like

<img width="884" alt="Screen Shot 2021-06-11 at 2 16 37 PM" src="https://user-images.githubusercontent.com/35148959/121738301-a920de80-cabf-11eb-930f-4abb2f99a2fd.png">